frontend/lib/src/components/core/LibContext.tsx

```diff
@@ -51,6 +51,13 @@ export interface LibContextProps {
    * @see ThemeCreatorDialog
    */
   addThemes: (themes: ThemeConfig[]) => void
+
+  /**
+   * Whether or not to hide full screen for all elements / widgets
+   * this is to get rid of unnecessary scroll bars in @streamlit/lib use cases
+   * will default to false for regular streamlit
+   */
+  hideFullScreenButtons?: boolean
```

Comment:
It's not optional, right? Just needs to be added to `App.tsx` explicitly as `false`, right?

Reply:
I added it as optional since Maya wanted it to remove it from `App.tsx`. Let me know if you want me to readd it.

Reply:
Just didn't want it integrated into `App.tsx`'s state since its not really stateful